packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -30,19 +31,26 @@ function BlockListAppender( {
 	rootClientId,
 	canInsertDefaultBlock,
 	isLocked,
-	renderAppender: CustomAppender,
+	renderAppender: CustomAppender, // Deprecated.
+	appender,
 	className,
 	selectedBlockClientId,
 	tagName: TagName = 'div',
 } ) {
-	if ( isLocked || CustomAppender === false ) {
+	if ( undefined !== CustomAppender ) {
+		deprecated( 'renderAppender prop', {
+			since: '10.9',
+			alternative: 'appender prop',
+		} );
+	}
+
+	if ( isLocked || appender === false || CustomAppender === false ) {
```

Comment:
Could we take this chance to deprecate `false` as a value too? This always seem a little bit weird to me while we can always just use `null` for the same thing.

Reply:
Hey Kevin! Thank you for checking!

I agree with you, but I think it would be a bigger breaking change. It would break sites that currently use `null` because any falsy (except the `false`) value runs the `else`. So I think it could be done in a future big change (maybe when changing this component for something else?). WDYT?

Reply:
Thank you for checking! =)
Hum. This makes sense! I'll take a look in the next few hours!

Reply:
This was implemented here: https://github.com/WordPress/gutenberg/pull/29911/commits/2537416fac2a8aba94a6160cc78a7caf16b4b453

I also added to the PR description, but copying the details here:

If the `appender` is `null`, nothing is rendered.
If the `appender` is `undefined` the default is rendered.
If the `appender` has any other value, it will be rendered.

Thank you again for the suggestion @kevin940726 !

Reply:
Just adding that I thought to not render anything for any falsy value instead of `null` only, but it wouldn't make sense for `0`, for example.
For other falsy (except `undefined` and `null`) it will render the `<TagName>` with the falsy value inside (the React handles if it should be rendered or not).